tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot

```diff
@@ -0,0 +1,102 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test OVA-installer
+Resource  ../../resources/Util.robot
+Suite Setup  Setup Suite Environment
+Suite Teardown  Cleanup Suite Environment
+
+*** Variables ***
+${dhcp}  True
+${ova_options}  --acceptAllEulas --name=vic-unified-ova-integration-test --X:injectOvfEnv --X:enableHiddenProperties -st=OVA --powerOn --X:waitForIp --noSSLVerify=true -ds=datastore1 -dm=thin  --prop:appliance.root_pwd="password"   --prop:appliance.permit_root_login=True   --prop:management_portal.port=8282   --prop:registry.port=443
+${esx_number}  1
+${not_collect_log}  False
+
+*** Keywords ***
+Setup Suite Environment
+    @{vms}=  Create A Simple VC Cluster  datacenter  cls  ${esx_number}  ${network}
+    ${vm_num}=  Evaluate  ${esx_number}+1
+    ${vm-names}=  Get Slice From List  ${vms}  0  ${vm_num}
+    ${vm-ips}=  Get Slice From List  ${vms}  ${vm_num}
+    Set Suite Variable  ${vc-ip}  @{vm-ips}[${esx_number}]
+    Set Suite Variable  ${vm-names}  ${vm-names}
+    Set Suite Variable  ${vm-ips}  ${vm-ips}
+    GOVC Use VC Environment
+
+Cleanup Suite Environment
+        Nimbus Cleanup  ${vm-names}  ${not_collect_log}
+
+GOVC Use VC Environment
+        Set Environment Variable  GOVC_URL  ${vc-ip}
+        Set Environment Variable  GOVC_USERNAME  Administrator@vsphere.local
```

Comment:
Username and password should be set by environment variable or consistent with the other manual tests